.jenkins/actions/run_standalone.sh

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+
+# Jenkins action to run a benchmark of dynamics.py on Piz Daint
+# 3/11/2021, Tobias Wicky, Vulcan Inc
+
+# stop on all errors and echo commands
+set -e
+
+# utility function for error handling
+exitError()
+{
+    echo "ERROR $1: $3" 1>&2
+    echo "ERROR     LOCATION=$0" 1>&2
+    echo "ERROR     LINE=$2" 1>&2
+    exit $1
+}
+
+# check arguments
+DO_PROFILE="false"
+SAVE_CACHE="false"
+SAVE_TIMINGS="false"
+if [ "$1" == "profile" ] ; then
+    DO_PROFILE="true"
+fi
+if [ "$1" == "build_cache" ] ; then
+    SAVE_CACHE="true"
+fi
+# only save timings if this is neither a cache build nor a profiling run
+if [ "${SAVE_CACHE}" != "true" -a "${DO_PROFILE}" != "true" ] ; then
+    SAVE_TIMINGS="true"
+fi
+
+# configuration
+SCRIPT=`realpath $0`
+SCRIPTPATH=`dirname $SCRIPT`
+ROOT_DIR="$(dirname "$(dirname "$SCRIPTPATH")")"
+FORTRAN_SERIALIZED_DATA_VERSION=7.2.5
```

Comment:
A lot of words for a very minor question:
Now that this is consolidated to one spot (yay!) for performance runs, do we want to keep the data version assigned specifically here, or grab it from the Makefile like .jenkins/jenkins.sh does here
export FORTRAN_VERSION=`grep "FORTRAN_SERIALIZED_DATA_VERSION=" Makefile  | cut -d '=' -f 2`
The risk of leaving it here is that we might forget to update it when we change the Makefile version. The risk of using the Makefile version is that we might forget to set up the c128 data in the project directory location and this will fail. The latter makes us stay up to date, the former allows us consistency with past runs, unless the data changes so much there are no longer the expected variables available for running the standalone. I don't mind either way, just wanted to bring it up.

Reply:
Great suggestion. Done.